CA_DataUploaderLib/ServerUploader.cs

```diff
@@ -558,6 +559,38 @@ public static async Task<PlotConnection> Establish(string loopName, byte[] publi
                 }
             }
 
+            private static async Task<(int plotId, string plotName)> GetPlotIDAsyncWithRetries(string loopName, HttpClient client, string loginToken, byte[] publicKey, byte[] signedVectorDescription, CancellationToken cancellationToken)

```

Comment:
Should we also add this to master? https://github.com/copenhagenatomics/CA_LoopCode/issues/790

Reply:
Yes, I would say the only bit of the PR that I would not merge is the change to ConnectTimeout **and** the catching of Exception for retries. On the later I also read they changed exceptions for HttpClient so it all goes to to the same exceptions type with a reason